SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -74,6 +87,89 @@ bool USpatialClassInfoManager::ValidateOrExit_IsSupportedClass(const FString& Pa
 	return true;
 }
 
+#if UE_EDITOR
+struct FLocalPopupInfo
+{
+	void OpenObject() const
+	{
+		const FSoftObjectPath ObjectAtPath(ObjectPath);
```

Comment:
This is never used?

Reply:
No, it's used in the next line and inside of the next check - `UObject* ObjectToOpen = ObjectAtPath.ResolveObject();`. Naming is somewhat confusing, though - I'll think on how to improve this.